src/apps/content-editor/src/app/components/Editor/Field/Field.js

```diff
@@ -129,17 +129,26 @@ function resolveRelatedOptions(
       );
     })
     .map((itemZUID) => {
-      return {
-        filterValue: items[itemZUID].data[field.name],
-        value: itemZUID,
-        component: (
-          <RelatedOption
-            modelZUID={modelZUID}
-            itemZUID={itemZUID}
-            text={items[itemZUID].data[field.name]}
-          />
-        ),
-      };
+      for (const [key, value] of Object.entries(items[itemZUID].siblings)) {
+        if (items[itemZUID].meta.ZUID === value) {
+          return {
+            filterValue: items[itemZUID].data[field.name],
+            value: itemZUID,
+            component: (
+              <RelatedOption
+                modelZUID={modelZUID}
+                itemZUID={itemZUID}
+                text={
```

Comment:
Lets rename this to `html` and change the reference in the `ResolvedOption` component as well. So whoever reads this later will have an easier time understanding the context.